spring-cloud-dataflow-dependencies/pom.xml

```diff
@@ -110,6 +110,20 @@
 				<artifactId>spring-cloud-dataflow-platform-cloudfoundry</artifactId>
 				<version>2.3.0.BUILD-SNAPSHOT</version>
 			</dependency>
+			<dependency>
+				<groupId>org.springframework.security</groupId>
+				<artifactId>spring-security-bom</artifactId>
+				<version>5.2.0.RELEASE</version>
```

Comment:
We need to find an alternative way to fix this as this way to explicit declaration might become stale after a few upcoming releases and will be painful to manually manage.